features/steps/steps.py

```diff
@@ -847,6 +906,120 @@ def stdout_matches_the_json_schema(context, schema):
         )
 
 
+@then("`{file_name}` is not present in any docker image layer")
+def file_is_not_present_in_any_docker_image_layer(context, file_name):
+    when_i_run_command(context, "ls -1R /var/lib/docker/overlay2", "with sudo")
```

Comment:
Since we are already running a shell command here, why not using `'find /var/lib/docker/overlay2 -name {}.format(file_name)'` and checking if anything was pushed to stdout (no need for regular expressions)?

Reply:
yes that is a much better approach, for some reason I never think to use `find` haha, will do